app/styles/_projects.less

```diff
@@ -43,7 +43,9 @@
       margin-left: 5px;
     }
     a {
+      display: block;
       line-height: initial;
+      .word-break-all();
```

Comment:
I'm def curious why that preference (not having a strong opinion myself, but impulse I'd prob tend to choose word-break-all).